db/migrate/20230714030255_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+    def change
+      create_table :likes do |t|
+        t.references :authors, null: false, foreign_key: true
+        t.references :posts, null: false, foreign_key: true
```

Comment:
- [ ] As specified in the ER diagram the column name inside the `likes` table should be `user_id` and `post_id` but the current migration file will create a column of `authors_id` and `posts_id` so I'd suggest you update them as shown below
```rb
    t.references :user, foreign_key: true
    t.references :post, foreign_key: true
```
- [ ] Since a similar pattern exists in the migration for the `comments` table I'd suggest you update it as well.